### PR TITLE
Amended too long lines in conditionset.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hi. 

I rectified couple of issues with file sympy/sets/conditionset.py:

1. There were several too long lines. These have been now amended.
2. There was unused import in one of the examples. This has been now removed. Example was tested locally after removal of unused import - fully functional.

./bin/test quality executed locally - all green.

Thank you.

### CODE REVIEW

File: sympy/sets/conditionset.py [hunk 1]
```diff
@@ -39,10 +39,20 @@ def __new__(cls, sym, condition, base_set):
         if isinstance(base_set, EmptySet):
             return base_set
         if isinstance(base_set, FiniteSet):
-            sifted = sift(base_set, lambda _: fuzzy_bool(condition.subs(sym, _)))
```
Comment (smichr) [inline sympy/sets/conditionset.py]: We tend to not prettify these too much. How about the following?

```
                sifted = sift(base_set, lambda _: fuzzy_bool(
                    condition.subs(sym, _)))
                return Union(FiniteSet(*sifted[True]), Basic.__new__(
                    cls, sym, condition, FiniteSet(*sifted[None])))
```

Comment (Newman101) [inline sympy/sets/conditionset.py]: Hi. Sure, I will fix this.

**Comment by debugger22** [comment]
@Newman101 Maybe this can be merged once you change what was suggested.

**Comment by Newman101** [comment]
Sure, I will do these next.

**Comment by Newman101** [comment]
Took a while but it's done now - both tests ran locally, all green.

**Comment by gxyd** [comment]
I had restarted the failing test, it has passed now. Thanks, I am merging this.

**Comment by Newman101** [comment]
Good to know, thanks!

**Pull request merged**